src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -403,6 +407,32 @@ function AuthScreens({session, lastOpenedPublicRoomID, initialLastUpdateIDApplie
         // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps
     }, []);
 
+    const clearStatus = () => {
+        User.clearCustomStatus();
```

Comment:
@nkdengineer This method will send `CLEAR_STATUS` command to the backend. Should we just update the optimistic data of frontend? See the offline update conflicts discussed here https://github.com/Expensify/App/issues/53518#issuecomment-2523516506

@rlinoz Can you help add pusher event for the expired status?

Reply:
I was testing this the other day and I think we already have it? Isn't it working?